README.md

```diff
@@ -4,13 +4,18 @@
 
 1. clone repo
 2. set env variables in `.bash_profile`, `.zshrc` or equivalent
-   ```
-   export ASPNETCORE_ENVIRONMENT=Development
-   export ASPNETCORE_HTTPS_PORT=5000
-   export NODE_ENV=development
-   ```
+    ```
+    export ASPNETCORE_ENVIRONMENT=Development
+    export ASPNETCORE_HTTPS_PORT=5001
+    export NODE_ENV=development
+    ```
 3. make init script executable `chmod +x init.sh`
 4. run init script `./init.sh`
+5. The init script builds, installs, and runs the client server and API server after initialization
```

Comment:
That's fair, I've never run an shell file from another shell file, do I just type `./start.sh`? I also have `fstart.sh` which is just running the project since everything else is built and running. Does the dotnet project doesn't need to be rebuilt every change?